x-pack/legacy/plugins/canvas/public/components/datasource/datasource_selector.js

```diff
@@ -6,7 +6,25 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { EuiCard, EuiIcon } from '@elastic/eui';
+import { EuiCard, EuiIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+const icons = ['essql', 'esdocs', 'demodata', 'timelion'];
+
+const cardNodes = icons.map(function(item, index) {
```

Comment:
Looks like this is leftover from testing something maybe? Delete if it's not necessary.

Reply:
good catch, thanks!